float64 new_x
float64 new_y
float64 new_theta
---
bool service_status